airflow/Makefile

```diff
@@ -14,3 +14,7 @@ black:
 
 pdoc: $(shell readlink -m $(shell dirname $(firstword $(MAKEFILE_LIST))))/image-release.marker
 	docker run -u 0:0 -v $(pwd)/opt/providers:/opt/airflow/providers --rm $(baseTag) bash -c '$$@' -- pdoc -o /opt/airflow/providers/etna/docs etna
+
+release::
```

Comment:
Is this the right way to auto-build `pdoc` on release?

Reply:
This, could work?  But due to the base release definition:

```
.PHONY: release
release:: release-build release-test ## Builds static docker images staged for release, runs tests against them, and pushes them to dockerhub (requires PUSH_IMAGES=1)
	@ true
```

It's possible that the new target runs after release-build and executes an additional release-build, or something.  Basically, I'm not 100% the Make behavior here when merging targets where the same target exists twice.

How about making it `release-build::` ?

Reply:
Sorry, very much so a Make noob -- changing it to

```
release-build::
	$(MAKE) pdoc
```

but I'm not sure the order will be right? It does run `pytest` and then `pdoc`, but then doesn't seem to do anything else like build an image with the latest docs? Do I need to add some sort of `super` command to the `release-build::` to call the regular build process after `$(MAKE) pdoc`?

Reply:
AH!  Ok.  Yeah, you are correct actually.  Let me see...

Reply:
Ok, alternative fix, while silly, is at the VERY top of the file, add:

```
release::
<tab>$(MAKE) pdoc
```

Before the other includes.   The multitarget `::` executes in order of definition, so we need this step to proceed the other release definition to inject it before the build image.

This works similar to the way you've written it, but I worry about Make logic invoking `release-build` twice as written.

Reply:
Okay, I think this works as expected. The final output I get is:

```
set -e; if ! [ -z "" ]; then docker push airflow; touch /projects/ucsf/monoetna/airflow/image-release.marker; fi
make: Leaving directory '/projects/ucsf/monoetna/airflow'
```

so I think if I had included the right environment variables, it would have pushed the latest build with latest docs. :+1: